freemocap/core_processes/detecting_things_in_2d_images/mediapipe_stuff/mediapipe_skeleton_detector.py

```diff
@@ -10,21 +10,23 @@
 from skellycam.opencv.video_recorder.video_recorder import VideoRecorder
 from tqdm import tqdm
 
-from freemocap.system.paths_and_files_names import MEDIAPIPE_2D_NPY_FILE_NAME, ANNOTATED_VIDEOS_FOLDER_NAME
 from freemocap.core_processes.detecting_things_in_2d_images.mediapipe_stuff.mediapipe_skeleton_names_and_connections import (
     mediapipe_tracked_point_names_dict,
 )
 from freemocap.parameter_info_models.recording_processing_parameter_models import MediapipeParametersModel
+from freemocap.system.paths_and_files_names import MEDIAPIPE_2D_NPY_FILE_NAME, ANNOTATED_VIDEOS_FOLDER_NAME, \
+    MEDIAPIPE_POSE_WORLD_FILE_NAME
 
 logger = logging.getLogger(__name__)
 
 
 @dataclass
 class Mediapipe2dNumpyArrays:
```

Comment:
This whole branch has picked up a lot of confusion around the use of `mediapipe2d` vs `mediapipe3d` 

This Class is just one example - it can't called `2d` and be filled with `XYZ` stuff named `3d`

I think we should rename this to remove the `2d` and `3d` parts of names and then come up with some outer layer naming convention to differentiate data that came from "mediapipe processed an image" vs data that was triangulated/reconstructed... 


That ain't gonna happen today though - sorry!